pkg/rule/accessrule.go

```diff
@@ -312,6 +313,22 @@ func (t Target) ToAPIDetail() types.AccessRuleTargetDetail {
 	return at
 }
 
+var slackRegex = regexp.MustCompile("@slack-(C[\\w-]+) (@\\w+)?")
+
+// ExtractSlackMentions attempts to pull any slack tags from the rule description. Slack mentions are specified in the format
+// of @slack-CHANNEL_ID @USER. The @USER tag is optional
+func (r *AccessRule) ExtractSlackMentions() (channel string, user string) {
```

Comment:
I think we should implement the capability of getting a list of mentions, so a rule can notify multiple groups and users

Reply:
A single channel and multiple mentions? That makes sense to me :)

Reply:
I would say multiple channels.  We can repeat the same channel if needed.

so:
```
Example rule something

@slack-channel-1 @mention1 @slack-channel-1 @mention2 @slack-channel-1 @mention3 @slack-other-channel-1 @some-other-mention
```

Reply:
What's the use case? I don't want to add complexity if its never going to be used.

Reply:
The use case is clear: to send the notification to multiple channels and/or mentions.

Specific examples are rules that need to notify different slack groups for the same case. You can find several cases in the datadog monitors in the company, I can point you a pair. For instance, a group having a EMEA and US split, some big team with lots of on-call groups.

I can imagine also add in copy other on-call team to help with approvals or scopes/rules that spawns across different teams.

At first glance, it shouldn't be that complex. Likely enough to use [FindAllStringSubmatch](https://pkg.go.dev/regexp#Regexp.FindAllStringSubmatch) to then iterate the response to append the results on a slice. Then iterate that to send multiple messages, likely in parallel with a waitGroup as the DMs. 

But it is just a suggestion, we can validate this and/or do it later.

Reply:
Updated, I did have to change the format for tagging users/groups to simplify the parsing